changelog/7035.bugfix.rst

```diff
@@ -0,0 +1 @@
+Modified :func:`sunpy.io.special.read_srs` to correctly handle uppercase SRS files and supplementary sections occurring after the main data sections (I., IA, II).
```

Comment:
```
<unknown>:7: WARNING: py:func reference target not found: sunpy.io.special.read_srs
```
Unsure what the problem is, it is imported here at this level.

from sunpy.io.special.srs import read_srs - Works
import sunpy.io.special.read_srs - Does not work.